key-exchange/Cargo.toml

```diff
@@ -1,5 +1,10 @@
 [package]
 name = "tlsn-key-exchange"
+authors = ["TLSNotary Team"]
+description = "Implementation of the TLSN-specific key-exchange protocol"
```

Comment:
Let's use the phrase "TLSNotary-specific" since the less ambiguity in the public description, the better.